src/Make/MakeEln.php

```diff
@@ -35,7 +35,11 @@ class MakeEln extends MakeStreamZip
 
     public function __construct(protected ZipStream $Zip, AbstractEntity $entity, protected array $idArr)
     {
-        parent::__construct($Zip, $entity, $idArr);
+        // changelog setting only used in PDF
```

Comment:
These variables are semantically uninteresting, but are a way of being more explicit about what we try to express (rather than writing ` parent::__construct($Zip, $entity, $idArr, false, false);`). Keyword arguments would've been nice here.

Reply:
You are free to use keywourd arguments ;)